scripts/prepdocs.py

```diff
@@ -223,16 +230,16 @@ def create_sections(filename, page_map, use_vectors):
             "sourcefile": filename
         }
         if use_vectors:
-            section["embedding"] = compute_embedding(content)
+            section["embedding"] = compute_embedding(content, embedding_deployment)
         yield section
 
 def before_retry_sleep(retry_state):
     if args.verbose: print("Rate limited on the OpenAI embeddings API, sleeping before retrying...")
 
-@retry(wait=wait_random_exponential(min=1, max=60), stop=stop_after_attempt(15), before_sleep=before_retry_sleep)
-def compute_embedding(text):
+@retry(retry=retry_if_exception_type(openai.error.RateLimitError), wait=wait_random_exponential(min=1, max=60), stop=stop_after_attempt(15), before_sleep=before_retry_sleep)
```

Comment:
This was also retrying for AuthenticationError before, so I've made it more specific now. No sense retrying 15 times for an auth error.